deploy/prerequisites.yaml

```diff
@@ -0,0 +1,122 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
Shall we not include the manually created artifact. Use the actually built artifacts since when someone who would manually be building from the source after a change, will try to use this manually created file instead.